### PR TITLE
[bitnami/influxdb] Chart standardized

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/influxdb chart to be in line with the rest of the catalog.

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Review by Mauraza — APPROVED** [review]
Hi @migruiz4,

Thanks for the PR!!  LGTM!

File: bitnami/influxdb/values.yaml [hunk 1]
```diff
@@ -279,8 +303,56 @@ influxdb:
...
+  ## @param influxdb.extraEnvVarsSecret Name of existing Secret containing extra env vars for InfluxDB&trade; nodes
+  ##
+  extraEnvVarsSecret: ""
+  ## @param influxdb.extraVolumes Array of extra volumes to be added to the deployment (evaluated as template). Requires setting extraVolumeMounts
+  ##
+  extraVolumes: []
+  ## @param influxdb.extraVolumeMounts Array of extra volume mounts to be added to the container (evaluated as template). Normally used with extraVolumes.
+  ##
+  extraVolumeMounts: []
+  ## @param influxdb.containerPorts [object] Container ports
```
Comment (carrodher) [inline bitnami/influxdb/values.yaml:334]: In this way both values are shown in the table
```suggestion
  ## @param influxdb.containerPorts Container ports
```

File: bitnami/influxdb/values.yaml [hunk 2]
```diff
@@ -741,8 +979,26 @@ metrics:
...
+    ## @param metrics.service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+    ## If "ClientIP", consecutive client requests will be directed to the same mongos Pod
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+    ##
+    sessionAffinity: None
+    ## @param metrics.service.sessionAffinityConfig Additional settings for the sessionAffinity
+    ## sessionAffinityConfig:
+    ##   clientIP:
+    ##     timeoutSeconds: 300
+    sessionAffinityConfig: {}
```
Comment (carrodher) [inline bitnami/influxdb/values.yaml:1000]:
```suggestion
    ##     timeoutSeconds: 300
    ##
    sessionAffinityConfig: {}
```

File: bitnami/influxdb/values.yaml [hunk 3]
```diff
@@ -810,6 +1077,10 @@ persistence:
   ## @param persistence.size Size of data volume
   ##
   size: 8Gi
+  ## @param persistence.annotations Persistent Volume Claim annotations
+  ##
+  annotations: {}
+
```
Comment (carrodher) [inline bitnami/influxdb/values.yaml:1083]:
```suggestion
```

File: bitnami/influxdb/values.yaml [hunk 4]
```diff
@@ -613,10 +830,27 @@ collectd:
...
+    ## @param collectd.service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+    ## If "ClientIP", consecutive client requests will be directed to the same mongos Pod
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+    ##
+    sessionAffinity: None
+    ## @param collectd.service.sessionAffinityConfig Additional settings for the sessionAffinity
+    ## sessionAffinityConfig:
+    ##   clientIP:
+    ##     timeoutSeconds: 300
+    sessionAffinityConfig: {}
```
Comment (carrodher) [inline bitnami/influxdb/values.yaml:853]:
```suggestion
    ##     timeoutSeconds: 300
    ##
    sessionAffinityConfig: {}
```

**Review by carrodher — APPROVED** [review]

**Pull request merged**